clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -184,9 +184,8 @@
                                         Importance.HIGH,
                                         HEARTBEAT_INTERVAL_MS_DOC)
                                 .define(PARTITION_ASSIGNMENT_STRATEGY_CONFIG,
-                                        Type.STRING,
-                                        "range",
-                                        in("range", "roundrobin"),
+                                        Type.CLASS,
+                                        RoundRobinAssignor.class,
```

Comment:
I think this is the right default assignor, but just to be clear this is changing a default. We're still ok with this since we haven't released this yet, but are there any concerns with inconsistency with the old consumer?